rmgpy/rmg/input.py

```diff
@@ -341,6 +345,28 @@ def liquid_reactor(temperature,
                                  "entries of the same format")
             initialConcentrations[spec] = [Quantity(conc[0]), Quantity(conc[1])]
 
+    if inletVolumetricFlowRate or initialVolume or inletConcentrations or residenceTime:
+        if (inletVolumetricFlowRate or initialVolume or inletConcentrations) and residenceTime:
```

Comment:
Thanks for the comment! My original thought was to make the assumption of initial concentrations == inlet concentrations since it's usually the case for many industrial unit ops. I agree that having the optional inlet concentrations for CSTR can be beneficial to users too. I will modify the code.